src/frontend/komponenter/Brev/vedtakstabell/lagVedtakstabellBoutgifter.ts

```diff
@@ -22,32 +23,62 @@ const lagBeregningstabell = (
+                        <th style="width: 120px; word-wrap: break-word; ${borderStylingCompact}">Stønadsbeløp</th>
                     </tr>
                 </thead>
                 <tbody>
-                    ${lagRaderForVedtak(beregningsresultat)}
+               ${
+                   beregningsresultat?.inneholderUtgifterOvernatting
+                       ? lagRaderForVedtakMidlertidigOvernatting(beregningsresultat)
+                       : lagRaderForVedtakLøpendeUtgifter(beregningsresultat)
+               }
                 </tbody>
             </table>`;
 };
 
-const lagRaderForVedtak = (beregningsresultat?: BeregningsresultatBoutgifter): string => {
+const lagRaderForVedtakMidlertidigOvernatting = (
+    beregningsresultat?: BeregningsresultatBoutgifter
+): string => {
     if (!beregningsresultat) {
         return '';
     }
     return beregningsresultat.perioder
-        .map((periode) => {
-            const datoperiode = formaterIsoPeriodeMedTankestrek(periode);
-            const merutgift = formaterTallMedTusenSkille(periode.sumUtgifter);
-            const stønadsbeløp = formaterTallMedTusenSkille(periode.stønadsbeløp);
-            // const stjernemerktRad = periode.delAvTidligereUtbetaling ? '*' : '';
-            const asteriksForSatsendring = periode.makssatsBekreftet ? '' : '*';
+        .map((periode) =>
+            periode.utgifterTilUtbetaling
+                .filter((utgift) => !utgift.erFørRevurderFra)
+                .map((utgift) =>
+                    lagRadForVedtak(
+                        { fom: utgift.fom, tom: utgift.tom },
+                        utgift.utgift,
+                        utgift.tilUtbetaling
+                    )
+                )
+        )
+        .join('')
+        .replaceAll(',', '');
+};
 
-            return `<tr style="text-align: right;">
-                        <td style="text-align: left; ${borderStylingCompact}">${datoperiode}</td>
-                        <td style="${borderStyling}">${merutgift} kr</td>
-                        <td style="${borderStyling}">${stønadsbeløp} kr${asteriksForSatsendring}</td>
-                    </tr>`;
-        })
+const lagRaderForVedtakLøpendeUtgifter = (
+    beregningsresultat?: BeregningsresultatBoutgifter
+): string => {
+    if (!beregningsresultat) {
+        return '';
+    }
+    return beregningsresultat.perioder
+        .map((periode) => lagRadForVedtak(periode, periode.sumUtgifter, periode.stønadsbeløp))
         .join('');
 };
+
+const lagRadForVedtak = (datoperiode: Periode, merutgift: number, stønadsbeløp: number) => {
+    const datoperiodeString = formaterIsoPeriodeMedTankestrek(datoperiode);
+    const merutgiftString = formaterTallMedTusenSkille(merutgift);
+    const stønadsbeløpString = formaterTallMedTusenSkille(stønadsbeløp);
+    // const stjernemerktRad = periode.delAvTidligereUtbetaling ? '*' : '';
+    // const asteriksForSatsendring = periode.makssatsBekreftet ? '' : '*';
```

Comment:
Skal vi ikke vise `*` for satsendring lenger?

Reply:
Jo! Tenkte vi ikke hadde laget logikken for dette backend. Tok det tilbake nå:
940715d239e94495a39d9766846c5a09e677bf7d